src/feedback.js

```diff
@@ -259,10 +261,18 @@ var feedbackFactory = function feedbackFactory($container, config) {
                 ? this.config.timeout[this.config.level]
                 : this.config.timeout;
 
-            $closer.off('click').on('click', function(e) {
-                e.preventDefault();
-                self.destroy();
-            });
+            $closer
+                .off('click')
+                .on('click', function(e) {
+                    e.preventDefault();
+                    self.destroy();
```

Comment:
why not use arrow function? Then you'll be able to get rid of the `self` variable here.